apps/partner2/test/routes.coffee

```diff
@@ -27,7 +27,9 @@ describe 'Partner routes', ->
         @next.calledOnce.should.be.ok
         _.isUndefined(@next.args[0][0]).should.be.ok()
 
-    _.each ['institution', 'gellery_default', 'gallery_deprecated'], (layout) ->
+    # TODO Update this test when institutions are approved
+    # _.each ['institution', 'gellery_default', 'gallery_deprecated'], (layout) ->
+    _.each ['gellery_default', 'gallery_deprecated'], (layout) ->
```

Comment:
This is unit test for the `requireNewLayout` function so _I think_ the above line is indeed what we want? And [in the above tests](https://github.com/artsy/force/pull/451/files#diff-d8e41963d9cd63306ca5a87280d8c291R21) we will include `'institution'` to make sure it "nexts to the middleware in this route stack... ".

In addition, if you want to cover more, we could add another unit test for the `requireAdminIfInstitution` function.